Directory.Build.props

```diff
@@ -4,8 +4,4 @@
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <TreatSpecificWarningsAsErrors />
   </PropertyGroup>
-
-  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="all" />
-  </ItemGroup>
 </Project>
```

Comment:
Probably we can leave this? Not sure why, but VS on Windows still complaints about missing targeting packs:
```csharp
C:\Program Files\Microsoft Visual Studio\2022\Preview\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(1221,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.6.2 were not found
```

Reply:
Argh! I have amended that in 3f9593e1c70a37b57f9c5e01ef1284b12cf8cda9. I use Rider and not Visual Studio, that's why I missed this issue which is common apparently: https://github.com/dotnet/runtime/issues/68716